rfc/3/index.md

```diff
@@ -0,0 +1,309 @@
+historical reasons][ome-model], [version 0.4 of the OME-Zarr
+specification][ngff 0.4] [imposes][ngff 0.4 multiscales metadata] strict
+restrictions on the
+applications:
+
+> The length of "axes" must be between 2 and 5 and MUST be equal to the
+> dimensionality of the zarr arrays storing the image data (see
+> "datasets:path"). The "axes" MUST contain 2 or 3 entries of "type:space" and
+> MAY contain one additional entry of "type:time" and MAY contain one additional
+> entry of "type:channel" or a null / custom type. The order of the entries MUST
+> correspond to the order of dimensions of the zarr arrays. In addition, the
+> entries MUST be ordered by "type" where the "time" axis must come first (if
+> present), followed by the  "channel" or custom axis (if present) and the axes
+> of type "space". If there are three spatial axes where two correspond to the
+> image plane ("yx") and images are stacked along the other (anisotropic) axis
+> ("z"), the spatial axes SHOULD be ordered as "zyx".
+
+And:
+
+> Each "datasets" dictionary MUST have the same number of dimensions and MUST
+> NOT have more than 5 dimensions.
+
+These restrictions prevent users from converting existing
+datasets to OME-Zarr. For example, Zeiss .czi datasets [may contain][czi format
+dimensions] dimensions such as H, I, and V to store different phases,
+illumination directions, or views respectively. To say nothing of synthetic data
+that may contain "artificial" dimensions such as principal components or axes of
+other dimensionality reduction-techniques from many images.
+
+## Motivation
+
+In addition to the .czi datasets mentioned in the preceding paragraph, this
+section describes six dataset types that are currently impossible to represent
+in OME-Zarr:
+
+- in [electron backscatter diffraction (EBSD)][ebsd], a microscopy technique
+  common in materials science, a beam of electrons is scanned over a surface,
+  and for each (2D) position in the scan, a full 2D diffraction pattern is
+  recorded, resulting in a 4-dimensional data array.
+- from the diffraction patterns, it is possible to obtain an *orientation map*,
+  containing a 3D angle at each 2D position of the material.
+- the same principles apply to [diffusion tensor imaging][dti], where a
+  three-dimensional diffusion tensor is measured at each voxel.
+- it is common to compute Fourier transforms of 3D images. The datasets have
+  three dimensions but they are measured in *frequency*, not space.
+- when computing segmentations, one may use finer or coarser priors, resulting
+  in overlapping, equally valid segmentations, for example, of organelles at
+  one level, cells at another, and tissues at yet another. One common way to
+  store such a segmentation is to add a dimension for "coarseness".
+- computed spaces may have arbitrary dimensions related to the computation. For
+  example, in subtomogram averaging of [cryo electron tomography][CryoET],
```

Comment:
Where did you want CryoET to point, @jni?

Reply:
Oops, fixed! 😅